schemas/system/Language.schema

```diff
@@ -0,0 +1,4 @@
+Dictionary({
+  Name: { domain: 'Nomen', display: true, required: true, unique: true },
```

Comment:
It is possible to have different locales for the same language, thus I don't think this field should be unique.

Reply:
@belochub I agree, but we'll also need to change the way `Localization` refers `Language` because `{Name: 'english', Locale: 'eng' }` will be treated as different from `{Name: 'english', Locale: 'en' }`. Maybe we should stick to only one Locale for language, for example, ISO 639-3.

Reply:
`{ Name: 'United Kingdom English', Locale: 'en-GB' }` unique